ros2_ws/src/joy_controller/joy_controller/my_first_node.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+import rclpy
+from rclpy.node import Node
+
+# Create a class that inherits from the Node class
+# The __init__ method is the constructor of the class
+class MyNode(Node):
```

Comment:
This isn't a publisher or a subscriber. It just spins?

Reply:
At the moment this is just placeholder code which will eventually be publishing to the jot topic with controller input.